### PR TITLE
Fix #851, Error is raised instead of returning text [WiP]

### LINKED ISSUE #851: ZeroDivisionError: float division by zero in pagerank_weighted.py
Got ZeroDivisionError: float division by zero in **pagerank_weighted.py**

Input Text: BYAnonymousArabe

Error thrown at:  `summarize(text_to_sum, word_count=50)`

gensim error

I know word_count is 50 and I'm just giving a word. But ideally it should output that single word instead of throwing an error isn't it?

Please throw some light

### PR DESCRIPTION
**Pull request by metalaman**

### CODE REVIEW

**Comment by tmylk** [comment]
Please add a test that exception is raised

**Comment by tmylk** [comment]
Thanks for the improvement!

File: gensim/test/test_summarization.py [hunk 1]
```diff
@@ -97,7 +97,7 @@ def test_text_summarization_returns_input_on_single_input_sentence(self):
         # Keeps the first sentence only.
         text = text.split('\n')[0]
 
-        self.assertEqual(summarize(text),text)
+        self.assertRaises(ValueError,summarize,text)
```
Comment (piskvorky) [inline gensim/test/test_summarization.py]: PEP8: space after comma.

File: gensim/test/test_summarization.py [hunk 2]
```diff
@@ -87,7 +87,7 @@ def test_text_summarization_raises_exception_on_short_input_text(self):
         text = "\n".join(text.split('\n')[:8])
 
         self.assertTrue(summarize(text) is not None)
-
+
```
Comment (piskvorky) [inline gensim/test/test_summarization.py]: No trailing whitespace please.

File: gensim/test/test_wikicorpus.py [hunk 3]
```diff
@@ -31,7 +31,7 @@ def setUp(self):
 
 
     def test_get_texts_returns_generator_of_lists(self):
-        
+        logger.debug("Current Python Version is "+str(sys.version_info))
```
Comment (piskvorky) [inline gensim/test/test_wikicorpus.py]: PEP8: space around binary operators.

**Pull request merged**